### PR TITLE
HOCS-4080: simplify localstack queue creation

### PR DESCRIPTION
**Pull request by LiamMacP**
Add a setup-queues.sh script that will add create the queues using the
 aws CLI. This also adds the dlq and associates it with the main queue
 to stop message pollution.

Remove the queue creation from the LocalStack `awsSqsClient` bean as
this is now being handled through the setup script.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -17,10 +17,19 @@ services:
       DEFAULT_REGION: eu-west-2
 
 steps:
+  - name: setup localstack
+    image: quay.io/ukhomeofficedigital/localstack # need curl which is not in alpine
+    commands:
+      - ./config/localstack/setup-queues.sh
```
Comment (nickmcmahon01) [inline .drone.yml:23]: after all that, is it really that easy?

Comment (nickmcmahon01) [reply .drone.yml:23]: oh, because its in STEPS and not Services..we have access to it?

Comment (LiamMacP) [reply .drone.yml:23]: Services expose themselves through their name, so we can make sure it's healthy and then just create the queue. Bit upset that I have to have a step for it, but yes, simple really...

**Review by nickmcmahon01 — APPROVED** [review]
This is a much cleaner solution.

**Pull request merged**